data/doc_pages/en/installation.md

```diff
@@ -12,15 +12,86 @@ If it is not the case, please refer to :
 
 - Docker Compose : [docs.docker.com/compose/overview/](https://docs.docker.com/compose/overview/)
 
+- mkvirtualenvwrapper : [https://virtualenvwrapper.readthedocs.io/en/latest/index.html](https://virtualenvwrapper.readthedocs.io/en/latest/index.html)
+
```

Comment:
I would not recommend virtualenvwrapper in such a documentation, as it is too specific.
`python3 -m venv` is the most standard way to create a virtualenv in python 3.
People willing to use virtualenvwrapper can still use it if they know it and want it.

Reply:
You are right. 
I will change the doc for both of your comments.